velox/exec/StreamingAggregation.cpp

```diff
@@ -256,7 +279,14 @@ void StreamingAggregation::evaluateAggregates() {
     }
 
     if (isRawInput(step_)) {
-      function->addRawInput(inputGroups_.data(), rows, args, false);
+      if (function->clusteredInput() &&
+          function->supportAddRawClusteredInput()) {
+        VELOX_DCHECK(rows.isAllSelected());
```

Comment:
Does that mean we do not support masked aggregations with this optimization? Would it be possible to remove this restriction?